src/searchModal/scss/searchModal.scss

```diff
@@ -138,6 +127,27 @@ $classTreeVariables: (
         overflow: auto;
     }
 
+    .content-toolbar {
+        height: 2rem;
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-end;
+        align-items: stretch;
+        align-content: space-around;
+    }
+    .content-area {}
```

Comment:
Is this empty rule needed?

Reply:
Very good catch! I foresaw several rules, then refined them, and I left one dummy class behind...